packages/sitecore-jss-nextjs/src/components/NextImage.tsx

```diff
@@ -85,7 +85,7 @@ export const NextImage: React.FC<NextImageProps> = ({
   const loader = (otherProps.loader ? otherProps.loader : sitecoreLoader) as ImageLoader;
 
   if (attrs) {
-    return <Image loader={loader} {...imageProps} />;
+    return <Image alt="no-alt-set" loader={loader} {...imageProps} />;
```

Comment:
I would recommend setting `alt` to `""` (empty string) instead of `no-alt-set` as this value will get used and could be misleading (false sense of doing the right thing). Setting it to empty means the image becomes decorative.https://www.w3.org/WAI/tutorials/images/

Reply:
@pzi Makes sense for me, changed to empty string